api/python/cellxgene_census/src/cellxgene_census/_open.py

```diff
@@ -184,25 +184,24 @@ def open_soma(
     else:
         selected_mirror = mirrors[mirrors["default"]]  # type: ignore
 
+    # TODO: Consider raising exceptions instead of issuing warnings, possibly introducing a "strict" mode to control the
+    #  behavior
     try:
         description = get_census_version_description(census_version)  # raises
     except ValueError:
-        # TODO: After the first "stable" is available, this conditional can be removed (keep the 'else' logic)
-        if census_version == "stable":
-            description = get_census_version_description("latest")
-            api_logger.warning(
-                f'The "{census_version}" Census version is not yet available. Using "latest" Census version '
-                f"instead."
-            )
-        else:
-            raise ValueError(
-                f'The "{census_version}" Census version is not valid. Use get_census_version_directory() to retrieve '
-                f"available versions."
-            ) from None
-
-    if description["alias"]:
+        raise ValueError(
+            f'The "{census_version}" Census version is not valid. Use get_census_version_directory() to retrieve '
+            f"available versions."
+        ) from None
+
+    if description.get("flags", {}).get("retracted", False):
+        api_logger.info(
```

Comment:
IMHO, this should be a `warning` level log call

Reply:
agreed, changed!